packages/shared/src/components/filters/FollowingFilter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { IconSize } from '../Icon';
+import { OpenLinkIcon } from '../icons';
+import { useAuthContext } from '../../contexts/AuthContext';
+import Link from '../utilities/Link';
+
+export const FollowingFilter = (): JSX.Element => {
```

Comment:
```suggestion
export const FollowingFilter = (): ReactElement=> {
```

Reply:
Why do we even have this component, i don't really get it...

Reply:
I separated it so that the parent component didn't need to add the auth context hook just for the link. But I'm also fine with moving it back into the parent

Reply:
I mean more from ux perspective, what does it bring anyone this empty screen 🤣
https://dailydotdev.slack.com/archives/C07ATUPBPJ9/p1726728429332939

Reply:
Oh yeah, was wondering about that too 😬